Hyper/Game.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Character.Shaders;
+using Common;
+using Common.UserInput;
+using Hud.Shaders;
+using Hyper.Controllers;
+using Hyper.Shaders;
+using NLog;
+using OpenTK.Graphics.OpenGL4;
+using OpenTK.Mathematics;
+using OpenTK.Windowing.Common;
+using OpenTK.Windowing.GraphicsLibraryFramework;
+
+namespace Hyper;
+
+public class Game : IInputSubscriber
+{
+    public bool IsRunning = true;
```

Comment:
now when the window loses focus, the animations keep playing. Consider setting this to false when focus is lost

Reply:
will fix that. but also just for the future we have to stop using Stopwatch and just use UpdateFrame event times since we don't want the game state to change when it's paused/out of focus